adapters/socketio_berkeley.c

```diff
@@ -650,7 +652,11 @@ int socketio_setoption(CONCRETE_IO_HANDLE socket_io, const char* optionName, con
         }
         else if (strcmp(optionName, "tcp_keepalive_time") == 0)
         {
+#ifdef __APPLE__
```

Comment:
I think we should always use IPPROTO_TCP (per [http://pubs.opengroup.org/onlinepubs/9699919799/](http://pubs.opengroup.org/onlinepubs/9699919799/))

---
In reply to: [75430277](https://github.com/Azure/azure-c-shared-utility/pull/21#discussion_r75430277) [](ancestors = 75430277)